salt/modules/ciscoconfparse_mod.py

```diff
@@ -0,0 +1,443 @@
+
+    parent_regex
+        The regular expression to match the parent lines against.
+
+    child_regex
+        The regular expression to match the child lines against.
+
+    ignore_ws: ``False``
+        Whether to ignore the white spaces.
+
+    saltenv: ``base``
+        Salt fileserver environment from which to retrieve the file. This
+        argument is ignored when ``config_path`` is not a ``salt://`` URL.
+
+    Usage example:
+
+    .. code-block:: python
+
+        objects = __salt__['ciscoconfparse.find_objects_w_child'](config_path='https://bit.ly/2mAdq7z',
+                                                                  parent_regex='line con',
+                                                                  child_regex='stopbits')
+        for obj in objects:
+            print(obj.text)
+    '''
+    ccp = _get_ccp(config=config, config_path=config_path, saltenv=saltenv)
+    lines = ccp.find_objects_w_child(parent_regex, child_regex, ignore_ws=ignore_ws)
+    return lines
+
+
+def find_lines_w_child(config=None,
+                       config_path=None,
+                       parent_regex=None,
+                       child_regex=None,
+                       ignore_ws=False,
+                       saltenv='base'):
+    r'''
+    Return a list of parent lines (as text)  matching the regular expression
+    ``parent_regex`` that have children lines matching ``child_regex``.
+
+    config
+        The configuration sent as text.
+
+        .. note::
+            This argument is ignored when ``config_path`` is specified.
+
+    config_path
+        The absolute or remote path to the file with the configuration to be
+        parsed. This argument supports the usual Salt filesystem URIs, e.g.,
+        ``salt://``, ``https://``, ``ftp://``, ``s3://``, etc.
+
+    parent_regex
+        The regular expression to match the parent lines against.
+
+    child_regex
+        The regular expression to match the child lines against.
+
+    ignore_ws: ``False``
+        Whether to ignore the white spaces.
+
+    saltenv: ``base``
+        Salt fileserver environment from which to retrieve the file. This
+        argument is ignored when ``config_path`` is not a ``salt://`` URL.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' ciscoconfparse.find_lines_w_child config_path=https://bit.ly/2mAdq7z parent_line='line con' child_line='stopbits'
+        salt '*' ciscoconfparse.find_lines_w_child config_path=https://bit.ly/2uIRxau parent_regex='ge-(.*)' child_regex='unit \d+'
+   '''
+    lines = find_objects_w_child(config=config,
+                                 config_path=config_path,
+                                 parent_regex=parent_regex,
+                                 child_regex=child_regex,
+                                 ignore_ws=ignore_ws,
+                                 saltenv=saltenv)
+    return [line.text for line in lines]
+
+
+def find_objects_wo_child(config=None,
```

Comment:
What about `find_objects_no_child`? I'm not sure if that's better, but I think the `wo` is a tad confusing.